src/client/main.cpp

```diff
@@ -0,0 +1,30 @@
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <errno.h>
+#include <unistd.h>
+#include <syslog.h>
+#include <string.h>
+#include <memory>
+
+#include "daemon.h"
+#include "config.h"
+
+using namespace std;
+
+int main(int argc, char** argv)
+{
+  try
+  {
+    unique_ptr<Daemon> daemon = unique_ptr<Daemon>(new Daemon());
+    daemon->run();
```

Comment:
This run function doesn't have a stop logic. When is it gonna stop? This means the stop line right after this will never run